ui/src/app/store/iprange/types/base.ts

```diff
@@ -0,0 +1,19 @@
+import type { IPRangeType } from "./enum";
+
+import type { APIError } from "app/base/types";
+import type { Subnet, SubnetMeta } from "app/store/subnet/types";
+import type { TimestampedModel } from "app/store/types/model";
+import type { GenericState } from "app/store/types/state";
+import type { VLAN, VLANMeta } from "app/store/vlan/types";
+
+export type IPRange = TimestampedModel & {
+  comment: string;
+  end_ip: string;
+  start_ip: string;
+  subnet: Subnet[SubnetMeta.PK];
+  type: IPRangeType;
+  user?: string;
```

Comment:
This is always defined, just as an empty string if there is no user.
`data["user"] = "" if obj.user is None else obj.user.`